### ISSUE TITLE
Dashboard: "Open in editor" should be actual links

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#4379]
## Bug Description

Right now, the "Open  in editor" _links_ use `window.location.href`. For accessibility reasons, they should be actual hyperlinks.

## Expected Behaviour

I should be able to use the "Open in editor" links as actual links, for example to open them in a new tab.

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by csossi** [comment]
Verified in QA

Note: This means that users can ONLY edit their stories in new browser tabs. Editor opens in a new tab whether they select "OPEN IN EDITOR" on thumbnail or from the three-dot menu. They are unable to edit their story in a refreshed/existing window
